pkg/osquery/tables/dataflattentable/exec.go

```diff
@@ -53,7 +55,23 @@ func TablePluginExec(client *osquery.ExtensionManagerClient, logger log.Logger,
 		panic("Unknown data source type")
 	}
 
-	return table.NewPlugin(t.tableName, columns, t.generateExec)
+	generateFunc := t.generateExec
+
+	// if the binary doesn't exists, no-op the table's generate function.
+	if _, err := os.Stat(t.execArgs[0]); err != nil {
+		level.Info(t.logger).Log("msg", "unable to find binary to exec", "err", err)
+		generateFunc = errorGenerate(fmt.Sprintf("unable to find binary: %s", err))
+	}
+	return table.NewPlugin(t.tableName, columns, generateFunc)
```

Comment:
This seems wrong -- this should be part of `generate` not instantiation. (Binaries may change over the life of the process)

Reply:
👍 yea I can move it. I had it in `generate` initially, and moved it to the instantiation to avoid unnecessary calls. Probably not worth the need to reboot after installing nmcli for launcher to detect it.